### PR TITLE
Update documentation and mark HTTP client class as isolated

### PR DESCRIPTION
**Pull request by LakshanSS**
## Purpose
- Fix https://github.com/wso2-enterprise/choreo/issues/6678
- Fix https://github.com/wso2-enterprise/choreo/issues/7135
- Fix https://github.com/wso2-enterprise/choreo/issues/7136

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
JDK 11
SL Beta2

### CODE REVIEW

File: storageservice/modules/blobs/Module.md [hunk 1]
```diff
@@ -1,110 +1,40 @@
+## Overview
+This module allows you to access the Azure Blob service REST API through Ballerina.
+It provides you the capability to execute operations like listContainers, listBlobs, putBlob, deleteBlob etc. It also allows you to execute management operations such as createContainer and deleteContainer etc.
```
Comment (shaniR) [inline storageservice/modules/blobs/Module.md:3]: like -> such as.
We need to provide a more comprehensive description of the module. For e.g. what the Azure Blob Service REST API is, what it does, and why someone needs to use it.

File: storageservice/modules/blobs/Module.md [hunk 2]
```diff
@@ -1,110 +1,40 @@
...
-
-* Azure Storage Account https://docs.microsoft.com/en-us/learn/modules/create-azure-storage-account/
-
-* You need to get a Shared Access Key or one of the Access Keys from the Azure Portal.
-
-
-## Step1: Import the Azure Storage Blobs Ballerina Library
+### Step1: Import connector
 
 First, import the `ballerinax/azure_storage_service.blobs` module into the Ballerina project
```
Comment (shaniR) [inline storageservice/modules/blobs/Module.md:21]: Let's remove the word "First,"

File: storageservice/modules/blobs/Module.md [hunk 3]
```diff
@@ -113,78 +43,68 @@ If you are using one of the Access Key, use the follwing format.
...
+        accessKeyOrSAS: "ACCESS_KEY_OR_SAS",
+        accountName: "ACCOUNT_NAME",
         authorizationMethod: "accessKey"
     };
 ```
 
-## Step3: Initialize Azure Storage Blob Client 
-
-Create the BlobClient using the configuration you have created as shown above.
+Create the BlobClient using the blobServiceConfig you have created as shown above.
```
Comment (shaniR) [inline storageservice/modules/blobs/Module.md:52]: Shall we make this and the above point under bullet points?

File: storageservice/modules/files/Module.md [hunk 4]
```diff
@@ -1,89 +1,39 @@
+## Overview
+This module allows you to access the Azure Storage File REST API through Ballerina.
+It provides you the capability to execute operations like getFileList, getDirectoryList, createDirectory, directUpload and getFile etc using the FileClient. It also allows you to execute management operations such as createShare and deleteShare etc using the ManagementClient.
```
Comment (shaniR) [inline storageservice/modules/files/Module.md:3]: Let's add a comprehensive description of the module.

File: storageservice/modules/files/Module.md [hunk 5]
```diff
@@ -1,89 +1,39 @@
...
 
 ```ballerina
     import ballerinax/azure_storage_service.files as azure_files;
 ```
 
-## Step2: Create Azure File Service Configuration
+### Step2: Create a new connector instance
 
-Create the connection configuration using the Shared Access Signature or Access Key, base URL and account name.
+Create a `azure_files:AzureFileServiceConfiguration` with the obtained Shared Access Signature or Access Key, base URL and account name.
```
Comment (shaniR) [inline storageservice/modules/files/Module.md:29]: Create an

Comment (shaniR) [reply storageservice/modules/files/Module.md:29]: Shall we put them under bullet points?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,45 +1,62 @@
...
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-azure-storage-service.svg)](https://github.com/ballerina-platform/module-ballerinax-azure-storage-service/commits/master)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Connect to Azure Storage Services using Ballerina.
-
-# Introduction
-
-## What is Azure Storage Service
-
 [Azure Storage Service](https://docs.microsoft.com/en-us/azure/storage/common/storage-introduction) is a highly
```
[the remark this answers is not in the document]
Comment (LakshanSS) [reply README.md:8]: Done

File: README.md [hunk 7]
```diff
@@ -1,45 +1,62 @@
...
+### Setting up the prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
+
+   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+
+   * [OpenJDK](https://adoptopenjdk.net/)
 
-* Azure Account to Access Azure Portal https://docs.microsoft.com/en-us/learn/modules/create-an-azure-account/
+        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
```
[the remark this answers is not in the document]
Comment (LakshanSS) [reply README.md:28]: We are following this as a standard across all the connectors.

File: README.md [hunk 8]
```diff
@@ -1,45 +1,62 @@
...
-* Java 11 Installed
-Java Development Kit (JDK) with version 11 is required.
+### Building the source
 
-* Ballerina SL Alpha 5 Installed
-Ballerina Swan Lake Alpha 5 is required. 
+Execute the commands below to build from the source.
 
-* Shared Access Signature (SAS) or One of the Access Key for authentication
+1. To build the gradle project:
```
[the remark this answers is not in the document]
Comment (LakshanSS) [reply README.md]: Done. Will use this as the common way

File: README.md [hunk 9]
```diff
@@ -1,45 +1,62 @@
...
-| Azure Storage Service|     2019-12-12     |
+## Contributing to Ballerina
+As an open source project, Ballerina welcomes contributions from the community. 
 
+For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).
 
-## Azure Storage Service - Blobs
-Please check the [Blob Service Connector Documentation](https://github.com/ballerina-platform/module-ballerinax-azure-storage-service/blob/main/modules/blobs/README.md) to get more information about the Azure Storage Blob Service Connector.
+## Code of conduct
+All the contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
```
[the remark this answers is not in the document]
Comment (LakshanSS) [reply README.md]: Done

File: storageservice/modules/blobs/Module.md (hunk 1 above)
Comment (LakshanSS) [reply storageservice/modules/blobs/Module.md]: Done

File: storageservice/modules/blobs/Module.md (hunk 2 above)
Comment (LakshanSS) [reply storageservice/modules/blobs/Module.md]: Done

File: storageservice/modules/files/Module.md (hunk 4 above)
Comment (LakshanSS) [reply storageservice/modules/files/Module.md]: Done

File: storageservice/modules/files/Module.md (hunk 5 above)
Comment (LakshanSS) [reply storageservice/modules/files/Module.md]: Done

Comment (LakshanSS) [reply storageservice/modules/files/Module.md]: > Shall we put them under bullet points?

Yes. It makes sense to put them under bullet points. 
Done in all the similar places

File: storageservice/modules/blobs/Module.md (hunk 3 above)
Comment (LakshanSS) [reply storageservice/modules/blobs/Module.md]: Done

**Review by abeykoon — APPROVED** [review]

File: storageservice/modules/blobs/endpoint.bal [hunk 10]
```diff
@@ -21,20 +21,26 @@ import ballerina/xmldata;
 import ballerina/lang.'xml;
 import ballerina/log;
 
-# Azure Storage Blob Client Object.
+# Azure Storage Blob connector allows you to access the Azure Blob service REST API.
+# The Blob service offers the following three resources: the storage account, containers, and blobs.
+# This connector lets you execute operations against the storage account, containers, and blobs.
```
Comment (shaniR) [inline storageservice/modules/blobs/endpoint.bal:26]: against the above resources.

**Pull request merged**